typings/externs.d.ts

```diff
@@ -102,8 +102,17 @@ declare global {
       friendlyMessage?: string;
     }
 
+    /**
+     * A record of DevTools Debugging Protocol events.
+     */
+    export type DevtoolsLog = Array<Protocol.RawEventMessage>;
+
     export interface Trace {
       traceEvents: TraceEvent[];
+      metadata?: {
+        'cpu-family'?: number;
```

Comment:
just curious why you called out this but not the rest..

Reply:
also... worth pointing out that this metadata object has some really useful stuff..

![image](https://user-images.githubusercontent.com/39191/38584744-1cf1dfc4-3ccc-11e8-9213-d4c74cd3944b.png)

Reply:
we support saving `metadata` in `asset-saver`, so we want to put it in there, but we don't really use it anywhere and I don't know what's actually required in there vs optional. However if we just put a `metadata?: {}` it treats it as `any`. Putting a single property triggers excess property checking, so could signal "hey, add some types if you want to use this". Could also set to `never` or something for the same effect...

Reply:
lol ok